public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx

```diff
@@ -178,7 +177,7 @@ function PanelOptionsPaneComponent({ model }: SceneComponentProps<PanelOptionsPa
               />
             )}
           </div>
-          <ScrollContainer>
+          <ScrollContainer minHeight={isScrollingLayout ? 'max-content' : 0}>
```

Comment:
think we need this PanelDataPaneRendered as well to disable scrolling in side the tab below viz. 

An alternative to this hook => minHeight would be for ScrollContainer to have an option like "disableBelow" or some responsive way to disable scrolling that it can internally control via normal css media query. (vs having multiple useMedia hooks that control react props that intern control css).  

but since it's just here we are using this it's probably not woth to add that yet but something to keep in mind of this scenario keeps coming up in more places.